pandas/tests/base/test_value_counts.py

```diff
@@ -190,6 +190,14 @@ def test_value_counts_bins(index_or_series):
 
     assert s.nunique() == 0
 
+    # handle normalizing bins with NA's properly
```

Comment:
make a new test

Reply:
Just to make sure I understand: are you saying this is a badly written test, so I should make a different one?  Or are you saying add an additional test beyond this one?

Reply:
test_value_counts_bins is already doing too much. make this a separate test.

Reply:
done